index.html

```diff
@@ -58,12 +58,13 @@
         <br>
         <div>
             <br>
+            <h4>"Hello! Coding World"</h4>
             <h1>BRILLIANT CODERS INTERNATIONAL SHOW 2023</h1>
             <p>
```

Comment:
- [ ] Kindly as stated by the previous reviewer 😃 keep the original style of the home page because the pages should look almost identical to the original design, outside the content differences. The pages use the exact same colors, for text, backgrounds, borders, etc, from the [original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 🙏🏼

- Basic color swatch: #272A31, #EC5242, #D3D3D3

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/225841141-59d6a514-4104-4d06-a991-2d2c1f6eae1a.png)   |   ![image](https://user-images.githubusercontent.com/83360636/225841078-36d59941-181d-4c05-9f7d-6283dfb0368e.png) |